django_loci/geo/geocode.py

```diff
@@ -0,0 +1,41 @@
+from django.http import JsonResponse
+from django.views import View
+from django.utils.module_loading import import_string
+
+from .settings import DJANGO_LOCI_GEOCODER, DJANGO_LOCI_GEOCODE_API_KEY, DJANGO_LOCI_GEOCODE_DELAY, DJANGO_LOCI_GEOCODE_RETRIES
+
+from geopy.extra.rate_limiter import RateLimiter
+
+
+class GeoCodeView(View):
+    geocoder = import_string('geopy.geocoders.' + DJANGO_LOCI_GEOCODER)
+    geolocator = geocoder(user_agent="django_loci")
+    geocode = RateLimiter(geolocator.geocode, DJANGO_LOCI_GEOCODE_DELAY, DJANGO_LOCI_GEOCODE_RETRIES)
+    reverse = RateLimiter(geolocator.reverse, DJANGO_LOCI_GEOCODE_DELAY, DJANGO_LOCI_GEOCODE_RETRIES)
+
+    def get(self, request):
+        address = request.GET.get('address', '')
+        lat = request.GET.get('lat', 0)
+        lng = request.GET.get('lng', 0)
+        key = request.GET.get('key', None)
```

Comment:
remove this, we don't have to leak the API key to the browser, there's no need for that